src/helpers.coffee

```diff
@@ -4,13 +4,15 @@ module.exports =
   octoHelpers:
     createLink: (path) ->
       url.resolve(process.env.HUBOT_OCTO_ENDPOINT, path)
+    slugIt: (name) ->
+      name.toLowerCase().replace(/[\. \s]/g, '-')
   httpHelpers:
     getBody: (response) ->
       new Promise (resolve, reject) ->
         resolve response.body
     validateSuccess: (response) ->
       new Promise (resolve, reject) ->
-        if response.status == 200
+        if response.status >= 200 || response.status < 300
```

Comment:
Why not Just split this out into each request type.  For example success is a 200. Maybe validateCreated would be 201 ?